### ISSUE TITLE
Сrash when using campaign kit

### ISSUE THREAD

**Issue by filippovdev** [RadiusNetworks/campaignkit-ios#1]
Im using only campaignKit, sometimes app crash and I receive this message in HockeyApp logger: 

 **Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[__NSCFType proximityKit:didFailWithError:]: unrecognized selector sent to instance 0x124ec3dd0'**

I dont have even proximitiKit.framework in my project, why he crash and say something about proximityKit i cant understand.

iOS 9.3.1
Swift 2.2

**Comment by csexton** [comment]
@filippovdev What version of the SDK are you running?

**Comment by filippovdev** [comment]
@csexton  Sorry for late answer I had a vacation. I use latest version and this bug is still appears. To reproduce:
first I init campaignKit:

```
campaignKit = CKManager(delegate: self, andConfig: location.kitProperties())
campaignKit?.start()
```

later I need to switch to another Kit

```
 campaignKit?.stop()
 campaignKit = CKManager(delegate: self, andConfig: newLocation.kitProperties())
```

And after a couple seconds after init app is crashed with error Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[__NSCFType proximityKit:didFailWithError:]: unrecognized selector sent to instance 0x124ec3dd0'

Looks like stop() method not stop all started network processes and must be some extra check if proximityKit delegate not nil.

**Comment by csexton** [comment]
Thanks for the extra details, @filippovdev. I think your assessment is correct. can you confirm that this only happens when you are switching kits? 

Will try to reproduce here.

**Comment by filippovdev** [comment]
@csexton yes it happened only after I reinit CKManager.

```
campaignKit?.stop()
campaignKit = CKManager(delegate: self, andConfig: newLocation.kitProperties()) // crush after 2-3 seconds after init
```

**Comment by greencoder** [comment]
Sorry for the delay, @filippovdev. I sent a sample project that switches your kits to the email address from your account. Can you please see if it crashes? I was not able to reproduce.

**Comment by greencoder** [comment]
This is fixed in 1.3.2.
